test_extras/testrunners.py

```diff
@@ -127,14 +129,15 @@ def get_coverage_files(app_labels, ignore_dirs, ignore_files):
     return a list of all the python files that should be included in
     a coverage report for that test.
     """
-    from django.db.models import get_app
-
     ret = []
 
     for app_label in app_labels:
-        module = get_app(app_label)
-        dirname = os.path.dirname(module.__file__)
-        ret.extend(get_coverage_files_in_directory(dirname, ignore_dirs, ignore_files))
+        if app_label in settings.INSTALLED_APPS:
+            module = __import__(app_label, fromlist=[''])
```

Comment:
https://docs.python.org/2/library/functions.html#__import__ recommends against using `__import__` in everyday Python programming in favour of `importlib.import_module()`.

If you do need to use it (not sure if `importlib.import_module()` is available in Python 2.6) then it might make sense to pass `globals(), locals(), [], -1)` to it like the examples in the above link do.